src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Each Tagging object represents an adding or deleting of a tag 
+ * for a specific person that happened during that session.
+ */
+public class Tagging {
+    
+    private final Person person;
+    private final Tag tag;
+    private String operationType;
```

Comment:
think why `String` may not be the best data type for this particular field?
what would be more appropriate?